tools/extend/README.md

```diff
@@ -0,0 +1,46 @@
+# `check` CLI tool
+
+This binary is a thin wrapper around the `rtmr` library to
+extend the measurement into RTMR registers.
+
+The tool's input is the event log.
+
+The tool's output is an error or "Success".
+
+## Usage
+
+```
+./extend [options...]
+```
+
+### `-in`
+
+This flag provides the path to the event log to measure. Stdin is "-".
+
+### `-rtmr`
+
+This flag defines the RTMR index that will be extended. As specified in the Intel TDX specification, user space applications can only extend to RTMR2 or RTMR3.
+
+### `-quiet`
+
+If set, doesn't write exit errors to Stdout. All results are communicated through exit code.
```

Comment:
Maybe "If set, no errors are written to stdout. Results are communicated through exit code only"